### ISSUE TITLE
New Question: multinomial log reg

### ISSUE THREAD

**Issue by turkunov** [Open-Deep-ML/DML-OpenProblem#284]
Multinomial logistic regression (also referred to as softmax regression) is a classification method that generalizes logistic regression to multiclass problems. Imho the problem would be a great challenge, as it's supposed to require:
* calc knowledge for softmax's gradient derivation
* revision of cross-entropy loss
* solid foundations of numpy

I'm going to be taking the implementation of this one. Want to do a little brush up on some stuff myself :)

**Comment by turkunov** [comment]
Added in https://github.com/Open-Deep-ML/DML-OpenProblem/pull/288

fyi @moe18
